include/fixed_containers/fixed_set.hpp

```diff
@@ -436,4 +436,29 @@ constexpr typename FixedSet<K, MAXIMUM_SIZE, Compare, COMPACTNESS, StorageTempla
     return erase_if_detail::erase_if_impl(c, predicate);
 }
 
+/**
+ * Construct a FixedSet with its capacity being deduced from the number of items being passed.
+ */
+template <class K,
+          class Compare = std::less<K>,
+          fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness COMPACTNESS =
+              fixed_red_black_tree_detail::RedBlackTreeNodeColorCompactness::EMBEDDED_COLOR(),
+          template <class, std::size_t> typename StorageTemplate = FixedIndexBasedPoolStorage,
+          std::size_t SIZE,
+          fixed_set_customize::FixedSetChecking<K> CheckingType =
+              fixed_set_customize::AbortChecking<K, SIZE>>
+[[nodiscard]] constexpr FixedSet<K, SIZE, Compare, COMPACTNESS, StorageTemplate, CheckingType>
+make_fixed_set(const K (&list)[SIZE],
+               const Compare& comparator = {},
+               const std_transition::source_location& loc =
+                   std_transition::source_location::current()) noexcept
+{
+    FixedSet<K, SIZE, Compare, COMPACTNESS, StorageTemplate, CheckingType> set{comparator};
+    for (decltype(SIZE) i = 0; i < SIZE; ++i)
```

Comment:
`SIZE` -> `MAXIMUM_SIZE` to match the parent template.

Reply:
Done!